content/Software Support/Installation/Error-bad-CPU-type-in-executable.md

```diff
@@ -0,0 +1,47 @@
+---
+title: 'Error "bad CPU type in executable"'
+---
+
+Under certain conditions, startup or other operations of the Arduino development software might fail with a "`bad CPU type in executable`" error.
+
+This problem can be solved by one of the methods described below:
+
+## If your Mac computer has an Apple Silicon processor
+
+Some of the tools used by the Arduino development software are built for use on "Intel" processors. These can still be used on the [Apple Silicon](https://support.apple.com/en-us/HT211814) Mac computers thanks to a program named **Rosetta** that Apple provides.
```

Comment:
I added the quotes because this is the non-technical and non-standard term Apple uses to refer to the [x86-64 CPUs](https://en.wikipedia.org/wiki/X86-64). Even though they are **Intel™** brand, Apple is using the "Intel" term as a description of the architecture of the CPU rather than the manufacturer (x86-64 is not the only architecture of processors manufactured by Intel).

But I'm not overly attached to the quotes so I'm happy to remove them if you think it is better that way.

Reply:
For me, it's a bit confusing, since it's not apparent that we're citing Apple's use of the term in particular. If the distinction is to be made, it may be better to just explain it outright. 

What if this paragraph was changed to something like:

```suggestion
Some of the tools used by the Arduino development software are built for use on x86-64 processors, like those in Intel-based Macs. These can still be used on the [Apple Silicon](https://support.apple.com/en-us/HT211814) Mac computers thanks to a program named **Rosetta** that Apple provides.
```